redis/tests/test_basic.rs

```diff
@@ -1139,6 +1139,14 @@ fn test_zrandmember() {
     let results: Vec<String> = con.zrandmember(setname, Some(-5)).unwrap();
     assert_eq!(results.len(), 5);
 
+    if !ctx.use_resp3 {
```

Comment:
does this ever happen?
I meant to add a test where the context is created with resp3.

Reply:
i've added RESP3 to `make test` so it's valid